src/apps/explorer/components/TransanctionBatchGraph/index.tsx

```diff
@@ -303,16 +304,22 @@ function TransanctionBatchGraph({
   )
 
   useEffect(() => {
```

Comment:
ubernit: it would be probably nice to move all the effect to a custom hook to keep the component simpler

Reply:
Did a very basic refactor - simply moving the code to a hooks and utils file.
Not worth to spend more time on this right now IMO.